mycore-base/src/main/java/org/mycore/common/xml/MCRURIResolver.java

```diff
@@ -1234,7 +1237,9 @@ public Source resolve(String href, String base) throws TransformerException {
             try {
                 if (resolved != null) {
                     MCRSourceContent content = new MCRSourceContent(resolved);
-                    MCRContentTransformer transformer = MCRLayoutTransformerFactory.getTransformer(transformerId);
+                    MCRLayoutTransformerFactory factory = MCRConfiguration.instance()
+                        .getInstanceOf(TRANSFORMER_FACTORY_PROPERTY, TRANSFORMER_FACTORY_PROPERTY_DEFAULT);
```

Comment:
Its better for refactoring if you use MCRLayoutTransformerFactory.class.getName() here.

Reply:
changed